helm/credentiald-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: credentiald
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
+      - get
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    resourceNames:
+      - credentiald-configmap
+    verbs:
+      - get
```

Comment:
we don't need this right? the pod itself isn't accessing the cm via the api

Reply:
will check